src/v2/Apps/Consign/Components/GetPriceEstimate/ConsignArtistAutosuggest.tsx

```diff
@@ -12,17 +23,70 @@ export const ConsignArtistAutosuggest: React.FC = () => {
     suggestions,
   } = usePriceEstimateContext()
 
+  const tracking = useTracking()
+
+  const trackFocusedOnSearchInput = () => {
+    tracking.trackEvent(
+      focusedOnSearchInput({
+        context_module: ContextModule.priceEstimate,
+        context_owner_type: OwnerType.consign,
+      })
+    )
+  }
+
+  const trackSelectedItemFromSearch = (
+    suggestion: ConsignSearchSuggestion["node"]
+  ) => {
+    tracking.trackEvent(
+      selectedItemFromSearch({
+        context_module: ContextModule.priceEstimate,
+        context_owner_type: OwnerType.consign,
+        owner_id: suggestion.internalID,
+        owner_slug: suggestion.slug,
+        owner_type: OwnerType.artist,
+        query: searchQuery,
+      })
+    )
+  }
+
+  const trackSearchedWithNoResults = () => {
+    tracking.trackEvent(
+      searchedWithNoResults({
+        context_module: ContextModule.priceEstimate,
+        context_owner_type: OwnerType.consign,
+        query: searchQuery,
+      })
+    )
+  }
+
+  const debouncedTrackSearchWithNoResults = debounce(
+    trackSearchedWithNoResults,
+    100
+  )
+
+  useEffect(() => {
+    if (suggestions?.length === 0) {
+      debouncedTrackSearchWithNoResults()
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [suggestions])
+
   return (
     <Autosuggest
       suggestions={suggestions ?? []}
+      onSuggestionsClearRequested={x => x}
```

Comment:
What does this do?

Reply:
Noticed in the browser console that `react-autosuggest` will throw an annoying warning if this function isn't implemented